### DISCUSSION TITLE
Issue with Pino's transport on Windows using Deno 2 and TypeScript - no output is written to the log file

### DISCUSSION THREAD

**Discussion by cvermeer579** [denoland/deno#26150] [General]
I raised the problem with Pino,  but they replied: "Thank you for the report. This project targets the Node.js runtime. Please file issues with the runtime if you are not using Node.js".

I’m using Deno 2 with TypeScript on Windows, and I've encountered an issue when trying to log with Pino using a transport.
The following code works perfectly and logs to the debug console as expected:

```ts
import { pino } from 'pino';
export const logger = pino();

```

However, when I introduce the transport to log to a file, no output appears, and the log file is not created:
```ts
import { pino } from 'pino';
const transport = pino.transport({
    level: 'trace',
    target: 'pino/file',
    options: { destination: '.\\log.txt' },  // Using Windows path
});
export const logger = pino(transport);

```

No errors are thrown, but the expected log file doesn't appear, and no logging is written.

I am running Deno with the --allow-write flag, and Deno’s internal logger is correctly logging output. Is there any known issue with using Pino's transport functionality on Deno or Windows, or is there something specific I might be missing in my configuration?

Warm regards
Corrie

**Comment by marvinhagemeister** [comment]
This is a bug in Deno. Converted this discussion to an issue here https://github.com/denoland/deno/issues/26154
